docs/apache-airflow/start/local.rst

```diff
@@ -24,6 +24,8 @@ This quick start guide will help you bootstrap an Airflow standalone instance on
 
 .. note::
 
+   Successful installation requires a Python 3 environment. Airflow is currently tested with Python 3.6, 3.7, 3.8, and 3.9. 
```

Comment:
I’d remove mentioning specifically what versions are tested since it’s too tedious to update this with every release.

Reply:
Thanks. Removed versions from new commit.

Reply:
Does not seem to be removed.